client/imports/login/login.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input } from '@angular/core';
+import { NgForm }    from '@angular/common';
+ 
+@Component({
+  selector: 'login',
+  templateUrl: '/client/imports/login/login.html'
+})
+
+export class Login { 
+
+	clickedForget(){
+		forgetPassword();
```

Comment:
You should be able to implement these.  It is covered in the tutorial (http://www.angular-meteor.com/tutorials/socially/angular2/angular-material-and-custom-angular-auth-forms#customauthenticationcomponents)